loom/engine/loom2d/l2dScript.cpp

```diff
@@ -67,7 +67,7 @@ bool Loom2DNative::sInitialized = false;
 
 static Matrix *StaticMatrixConstructor(lua_State *L)
 {
-    return new Matrix((float)lua_tonumber(L, 2), (float)lua_tonumber(L, 3),
+    return lmNew(NULL) Matrix((float)lua_tonumber(L, 2), (float)lua_tonumber(L, 3),
```

Comment:
I'm not entirely sure what that does, but maybe it just allocates one and keeps it around until you kill the process?